src/processout/actionhandler.ts

```diff
@@ -201,7 +222,7 @@ module ProcessOut {
 
                 // Create the iframe to be used later
                 var iframe = document.createElement("iframe");
-                iframe.setAttribute("style", `margin: 1em auto; width: 440px; height: 480px; max-width: 100%; max-height: 100%; display: block; box-shadow: 0 15px 35px rgba(50, 50, 93, 0.1), 0 5px 15px rgba(0, 0, 0, 0.07); background-color: #ECEFF1; background-image: url("${this.instance.endpoint("js", "/images/loader.gif")}"); background-repeat: no-repeat; background-position: center;")`);
+                iframe.setAttribute("style", `margin: 1em auto; width: ${this.options.iframeWidth}px; height: ${this.options.iframeHeight}px; max-width: 100%; max-height: 100%; display: block; box-shadow: 0 15px 35px rgba(50, 50, 93, 0.1), 0 5px 15px rgba(0, 0, 0, 0.07); background-color: #ECEFF1; background-image: url("${this.instance.endpoint("js", "/images/loader.gif")}"); background-repeat: no-repeat; background-position: center;")`);
```

Comment:
just wondering what will be the impact for PO.js current users if we now dynamically change the width and height of the Iframe? Meaning that they will need to update their current implementation of the Iframe? maybe worth to set the old width and height in case where they don't provide their dimension ?

Reply:
The change is backward compatible: 

If options is set, the width and height will always be default to 400 * 480: https://github.com/processout/processout.js/pull/84/files/bb6ccbaf5a4691d02ba1475eea6d28358bbed118#diff-b309e160b9cac5a7d0a25558ef377593b10677ac20c36aea7dd40babcee34c24R75

Reply:
No problem, I tried to keep this change as less impactful as possible. It should allow us to get around the Safari popup issue also easy to maintain in the future.